docs/cow-protocol/concepts/how-it-works/flow-of-an-order.md

```diff
@@ -16,7 +16,7 @@ There are 4 main steps to an order on CoW Protocol:
 1. Users express their trade intents by signing messages that specify the assets and amounts they want to trade, alongside other parameters.
 2. The protocol gathers the intents of multiple users into a [batch auction](../introduction/batch-auctions).
 3. Solvers have a set amount of time to propose settlements for the batch. The solver that is able to generate the highest amount of surplus for the batch is declared the winner.
-4. The winning solver executes a single transaction on-chain, settling all orders in the batch on behalf of the users.
+4. The winning solver submits the batch transactions on-chain on behalf of the users.
```

Comment:
```suggestion
4. The winning solver submits the batch transaction on-chain on behalf of the users.
```

Nit: As talking about a "winning solver", it is a singular batch that's being discussed, as such, there can only be 1 batch transaction.